sys/include/net/netopt.h

```diff
@@ -375,6 +375,11 @@ typedef enum {
      */
     NETOPT_INTEGRITY_CHECK,
 
+    /**
+     * @brief   Get/Set the channel frequency as uint32_t.
```

Comment:
Maybe add some detailed description (i.e. paragraph under this line), about how this differs from the `NETOPT_CHANNEL` option.

Reply:
Ping @aabadie?

Reply:
Not sure if better, but I added the detailed description.

Reply:
Maybe not perfect, but sufficient for me.